doc/api/constants.md

```diff
@@ -0,0 +1,1089 @@
+# Constants
+
+The `constants` module provides convenient access to commonly used constants for
+use with file access, crypto methods, error handling and more. No methods are
+exposed by the `constants` module itself.
+
+**Note:** Specific constants may not be available on every operating system.
+
+## Error Constants
+
+The following error constants are adapted directly from the POSIX `errno`
+standards. See http://man7.org/linux/man-pages/man3/errno.3.html for more
+details about specific error constants.
```

Comment:
Can, but those are far from complete. We should either fill those out there more or move that detail into here. How those are documented leaves much to be desired.